### PR TITLE
Amazon Reviews Scraper for any product and putting it into a csv file

### PR DESCRIPTION
**Pull request by arjuaman**
Issue #92

Using scrapy, it scrapes the reviews from amazon.
Copy the link os the review page in `myBaseUrl` variable in amazon_reviews_scraping/spider/scraper.py

Run this in terminal to get the reviews in a csv file:
`scrapy runspider amazon_reviews_scraping/amazon_reviews_scraping/spiders/scraper.py -o reviews.csv`

### CODE REVIEW

File: Web-Scraping/amazon_reviews_scraping/spiders/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# This package will contain the spiders of your Scrapy project
```
Comment (Neilblaze) [inline Web-Scraping/amazon_reviews_scraping/spiders/__init__.py:1]: Shift these comments to a separate section, better would be in a README file, where you'll keep all your directions and installation guidelines upon this repo!

**Comment by Neilblaze** [comment]
Thank you for your contribution @arjuaman !

Before I approve changes to the PR, I will request you the following changes mentioned below.

- Add README (guide) along with this inside the folder same as your <filename>
- Shift most of the comments as instructions in the README file.
- Maintain proper [Contribution protocols](https://github.com/ankitdobhal/Awesome-Python-Scripts/blob/master/Contribution.md) (ignore if you're already following it!) & [good practices]((https://gist.github.com/sloria/7001839)!

Rest LGTM :+1:

**Comment by Neilblaze** [comment]
Thank you for your contribution @arjuaman !

Before I approve changes to the PR, I will request you the following changes mentioned below.

- Add README (guide) along with this inside the folder same as your `<filename>`
- Shift most of the comments as instructions in the README file.
- Maintain proper [Contribution protocols](https://github.com/ankitdobhal/Awesome-Python-Scripts/blob/master/Contribution.md) (ignore if you're already following it!) & [good practices](https://gist.github.com/sloria/7001839)!

Rest LGTM :+1:

File: Web-Scraping/amazon_reviews_scraping/spiders/__init__.py (hunk 1 above)
Comment (arjuaman) [reply Web-Scraping/amazon_reviews_scraping/spiders/__init__.py:1]: Thanks for the suggestions, on it!

**Comment by arjuaman** [comment]
Hey @Neilblaze , could you kindly take a look at it now? Thanks!

File: Web-Scraping/amazon_reviews_scraping/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+## USE: As a part of projects where you are required to do sentiment analysis on customer reviews data. </br>
+
+Scrapy is a web crawling framework for a developer to write code to create, which defines how a particular site (or a group of websites) will be scrapped.
+
+## Steps:</br>
```
Comment (Neilblaze) [inline Web-Scraping/amazon_reviews_scraping/README.md]: Use three sharps('#') in Steps section!

Rest LGTM :+1:

**Review by Neilblaze — CHANGES_REQUESTED** [review]

**Comment by Neilblaze** [comment]
I just noticed that you haven't created a new branch of your pr name within your forked repo! It's highly recommended to create a branch checkout before submitting PR! Follow [this](https://github.com/ankitdobhal/Awesome-Python-Scripts/blob/master/Contribution.md) if you're stuck.

& also, what is the role of `__init__.py` here as an empty file within the script directory?

**Comment by arjuaman** [comment]
`__init__.py` is system generated and is maybe updated temporarily at the time of scraping, so I left it as it is. @Neilblaze

File: Web-Scraping/amazon_reviews_scraping/README.md (hunk 2 above)
Comment (arjuaman) [reply Web-Scraping/amazon_reviews_scraping/README.md]: Done.

Comment (arjuaman) [reply Web-Scraping/amazon_reviews_scraping/README.md]: Should I close this pull request and open another from a branch?

**Comment by Neilblaze** [comment]
Cool! now LGTM :+1:

**Review by Neilblaze — APPROVED** [review]

**Review by Neilblaze — APPROVED** [review]

**Pull request merged**